libs/sr/dm-localization/src/executors/gen-locale/lib/hashData.ts

```diff
@@ -36,40 +37,6 @@ export const HashData = {
   lightCone: lightConeHashData,
   lightConeNames,
 
-  sheet: sheetHashData,
```

Comment:
One of the reason why we have separated namespace -> separated .JSON is that translations are usually lazy loaded. when the artifact page is loaded, it doesn't need any of the character/team UI translations, and vice versa. This can impact load time of pages and elements, since most of our elements are behind loading barriers and suspenses. we can designate some of the namespace to be permanently loaded(i think "ui" is).